SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialLoadBalanceEnforcer.cpp

```diff
@@ -46,15 +46,41 @@ void USpatialLoadBalanceEnforcer::OnAuthorityIntentComponentUpdated(const Worker
 	}
 }
 
+// This is called whenever this worker becomes authoritative for the ACL component on an entity.
+// It is now this worker's responsibility to make sure that the ACL reflects the current intent,
+// which may have been received before this call.
 void USpatialLoadBalanceEnforcer::AuthorityChanged(const Worker_AuthorityChangeOp& AuthOp)
 {
 	if (AuthOp.component_id == SpatialConstants::ENTITY_ACL_COMPONENT_ID &&
 		AuthOp.authority == WORKER_AUTHORITY_AUTHORITATIVE)
 	{
+		const AuthorityIntent* AuthorityIntentComponent = StaticComponentView->GetComponentData<SpatialGDK::AuthorityIntent>(AuthOp.entity_id);
+		if (AuthorityIntentComponent == nullptr)
+		{
+			// TODO(zoning): There are still some entities being created without an authority intent component.
+			// For example, the Unreal created worker entities don't have one. Even though those won't be able to
+			// transition, we should have the intent component on them for completeness.
```

Comment:
Discussion that might not need a change in this PR - I think more than completeness when we a better idea what multi worker is meant to mean that we will likely want to specify authority intents to know which things are grouped together / crash recovery etc.

Reply:
Yup, agreed, definitely something to look into in Jan when we start looking at actor sets.